Configuration/webapp/app/api/axiosInstance.ts

```diff
@@ -28,8 +28,9 @@ const axiosInstance = axios.create({
 
 axiosInstance.interceptors.request.use(async (config) => {
   const { token } = await getSessionData();
+  // elo
   if (token) {
-    config.url = `${config.url}?token=${token}`;
+    config.url = `${config.url}${config.url?.includes('?') ? '&' : '?'}token=${token}`;
```

Comment:
Consider utilising JS URL constructor:

```suggestion
    const url = new URL(config.url);
    url.searchParams.append('token', token);
    config.url = url.toString();
    ```